pages/how_tos/bitbucket.md.erb

```diff
@@ -30,8 +20,20 @@
 	* Merged
 	* Declined 
 
-Once you've entered all of this information, the form should look like this:
-<screenshot here>
+Once you've filled it out, the form should look like this:
+<%= image "webhook-form.png", width: 930/2, height: 1578/2, alt: "Screenshot of a completed Bitbucket webhook form" %>
+
+Click the `save` button, and the webhook will be created. Set up a Buildkite pipeline using this repository's url, and the repository events that you selected will trigger the creation of Buildkite builds 🎉
+
+As you set up a new Buildkite pipeline with a Bitbucket repository, we'll display these instructions again, populated with your unique account details. 
+
+## Connecting Buildkite to Bitbucket
+
+Connecting Bitbucket as an app in Buildkite will allow webhooks from Bitbucket access to your Buildkite account. Enabling this will allow Bitbucket's build status icons to reflect the state of your Buildkite builds. This step is optional - not adding Bitbucket as a connected app will not affect your ability to trigger Buildkite builds from your Bitbucket repository.  
```

Comment:
The special generated webhook URL includes a token which authenticates what Bitbucket sends to Buildkite without another "connect to Buildkite" step. 👮